bin/GameIndex.dbf

```diff
@@ -9607,12 +9668,15 @@ Serial = SLES-51227
 Name   = Tomb Raider - Angel of Darkness
 Region = PAL-M10
 Compat = 5
-[patches]
-	// 54AD76D7
-	comment=- This gamedisc supports multiple languages and widescreen.
-	comment=- Language & widescreen selection is done through the BIOS configuration.
-	comment=- Implementation requires Full boot - "Boot CDVD (full)".
-	comment=- Full boot is recommended anyway for this gamedisc to avoid textproblems.
+[patches = 54AD76D7]
+	author=Prafull
+	// Fix ingame SPS by interchanging vclipw.xyz vf5, vf5w with cfc2.
+	patch=1,EE,001db3a0,word,48489000
+	patch=1,EE,001db3a4,word,4bc529ff
+	// comment=- This gamedisc supports multiple languages and widescreen.
+	// comment=- Language & widescreen selection is done through the BIOS configuration.
+	// comment=- Implementation requires Full boot - "Boot CDVD (full)".
+	// comment=- Full boot is recommended anyway for this gamedisc to avoid textproblems. Needs DX11 to work properly
```

Comment:
Any reason why you commented these out?

Reply:
They are comments, i moved them below the patch. It's for formatting reasons and easier to see what is what.

Reply:
You disabled the comments from displaying on the console.